libraries/AP_OSD/AP_OSD.h

```diff
@@ -258,7 +264,14 @@ class AP_OSD
     };
 
     void set_nav_info(NavInfo &nav_info);
-
+    // disable the display
+    void clear() {
```

Comment:
Shouldn't this be `disable()`, then, with perhaps `clear()` as a side-effect?

Reply:
Ok